### DISCUSSION TITLE
Affiliate Plugin

### DISCUSSION THREAD

**Discussion by uditgoenka** [medusajs/medusa#8484] [Feature Requests]
Will be great to have an affiliate plugin inside Medusa for marketplace module, making it easy to onboard affiliates along with automated payout via Wise, PayPal and Stripe.

**Comment by radoslaw-sz** [comment]
hi @uditgoenka  - some very basic version of affiliate plugin you can find here: https://github.com/RSC-Labs/medusa-affiliate-discount (I am an author)
